index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <style>
+      body {
+        display: flex;
+        flex-direction: column;
+        background-color: #eee;
+      }
+
+      @media screen and (min-width: 200px) {
+        body {
+          background-color: #eee;
+        }
+      }
+    </style>
```

Comment:
I think it would be better to style your page in your external CSS file. Using separate files for your styles will make your code much more readable.